### PR TITLE
Dynamically limit the number of participants in a room

### PR DESCRIPTION
**Pull request by abora8x8**

### CODE REVIEW

File: resources/prosody-plugins/mod_muc_max_occupants.lua [hunk 1]
```diff
@@ -17,10 +18,12 @@ end
 
 local function check_for_max_occupants(event)
   local room, origin, stanza = event.room, event.origin, event.stanza;
-
-	local actor = stanza.attr.from;
   local user, domain, res = split_jid(stanza.attr.from);
 
+  if is_healthcheck_room(room.jid) then
```
Comment (damencho) [inline resources/prosody-plugins/mod_muc_max_occupants.lua:23]: Nice 👍

**Review by damencho — APPROVED** [review]

**Comment by damencho** [comment]
Jenkins, add to whitelist

**Review by damencho — APPROVED** [review]

**Comment by anirbandas18** [comment]
who calls the method `check_for_max_occupants(event)` as defined in **mod_muc_max_occupants.lua** script?

**Comment by damencho** [comment]
@anirbandas18 Please, when you have questions or problems use the community forum before opening new issues, thank you.

It is called here: https://github.com/jitsi/jitsi-meet/blob/master/resources/prosody-plugins/mod_muc_max_occupants.lua#L71
When that event is triggered by prosody.

**Pull request merged**